modules/network-observability-quickfilter.adoc

```diff
@@ -11,19 +11,22 @@ Query Options::
+** *Duplicated flows*: A flow might be reported from several interfaces, and from both source and destination nodes, making it appear in the data several times. By selecting this query option, you can choose to show duplicated flows. Duplicated flows have the same sources and destinations, including ports, and also have the same protocols, with the exception of `Interface` and `Direction` fields. Duplicates are hidden by default. Use the *Direction* filter in the *Common* section of the filters dropdown to switch between ingress and egress traffic.
 ** *Match filters*: You can determine the relation between different filter parameters selected in the advanced filter. The available options are *Match all* and *Match any*. *Match all*  provides results that match all the values, and *Match any* provides results that match any of the values entered. The default value is *Match all*.
 ** *Limit*: The data limit for internal backend queries. Depending upon the matching and the filter settings, the number of traffic flow data is displayed within the specified limit.
 
 Quick filters::
 The default values in *Quick filters* drop-down menu are defined in the `FlowCollector` configuration. You can modify the options from console. 
 
 Advanced filters::
-You can set the advanced filters by providing the parameter to be filtered and its corresponding text value. The section *Common* in the parameter drop-down list filters the results that match either *Source* or *Destination*. To enable or disable the applied filter, you can click on the applied filter listed below the filter options.
+You can set the advanced filters, *Common*, *Source*, or *Destination*, by selecting the parameter to be filtered from the dropdown list. The flow data is filtered based on the selection. To enable or disable the applied filter, you can click on the applied filter listed below the filter options.
 
 [NOTE]
 ====
 To understand the rules of specifying the text value, click *Learn More*.
 ====
 
-You can click *Reset default filter* to remove the existing filters, and apply the filter defined in `FlowCollector` configuration.
\ No newline at end of file
+You can toggle between image:arrow-up-long-solid.png[,10] *One way* and image:arrow-up-long-solid.png[,10] image:arrow-down-long-solid.png[,10] *Back and forth* filtering. The image:arrow-up-long-solid.png[,10] *One way* filter shows *Source* or *Destination* traffic according to your filter selections. The image:arrow-up-long-solid.png[,10] image:arrow-down-long-solid.png[,10] *Back and forth* filter includes return traffic to your existing *Source* or *Destination* filters.
```

Comment:
Is it worth mentionning that you can **swap** `Source` and `Destination` ? 
That's useful using one way.

![image](https://github.com/openshift/openshift-docs/assets/91894519/f71f3c47-9b49-4270-8f2e-02d94dff2524)
![image](https://github.com/openshift/openshift-docs/assets/91894519/6d2641b7-d245-4669-9292-92bf89a0234d)

Reply:
I agree, we should mention it and the utility of using it when filtering one way.